drivers/base/kdgdb/kdgdb.h

```diff
@@ -19,7 +19,7 @@
 
 #include <pstypes.h>
 
-// #define KDDEBUG /* uncomment to enable debugging this dll */
+#define KDDEBUG /* uncomment to enable debugging this dll */
```

Comment:
Maybe comment this out again when finalizing the PR?

Reply:
Sure